test/data/parsedSinglePenalties.js

```diff
@@ -2,9 +2,10 @@ export default [
   {
     complete: true,
     reference: '0122451124578',
-    enabled: true,
+    enabled: false,
     paymentCode: '5ef305b89435c670',
     issueDate: '09/03/2018',
+    dateTime: 1520553600,
     vehicleReg: 'SD45TXT',
     formattedReference: '12245-1-124578-IM',
     location: 'Dalar Hir (A55 near Holyhead)',
```

Comment:
Might want to review locations, as we might not have made it public knowledge where these sites are

Reply:
The locations are already in the codebase. We can raise a seperate task to review test data.